Atomic/syscontainers.py

```diff
@@ -764,25 +765,32 @@ def _check_system_oci_image(self, repo, img, upgrade):
         manifest = self._skopeo_get_manifest(img)
         layers = SystemContainers.get_layers_from_manifest(manifest)
         missing_layers = []
+        missing_gomtree_manifests = []
         for i in layers:
             layer = i.replace("sha256:", "")
             if not repo.resolve_rev("%s%s" % (OSTREE_OCIIMAGE_PREFIX, layer), True)[1]:
```

Comment:
could we combine here the two conditions and not require `missing_gomtree_manifests`?
```
has_layer = repo.resolve_rev("%s%s" % (OSTREE_OCIIMAGE_PREFIX, layer), True)[1]
has_gomtree_manifest = os.path.isfile(os.path.join(ATOMIC_VAR, "gomtree-manifests/%s.mtree" % layer)):
if not has_layer or not has_gomtree_manifest:
    missing_layers.append(layer)
    util.write_out("Missing layer %s" % layer)
```